packages/manager/modules/advices/src/advices/index.less

```diff
@@ -0,0 +1,20 @@
+.oui-advice-tile oui-tile-button.oui-tile__item {
```

Comment:
Instead of using `.oui-advice-tile` maybe we can scope to component ?

Reply:
Do you mean to add this class to the component? do you think it will not bring any advantages/improvements?

Reply:
No what I meant is instead of creating a class, scope the style to the component
```suggestion
ovh-advices {
 oui-tile-button.oui-tile__item {
```